tutorials/multi-tenant-tutorial.rst

```diff
@@ -0,0 +1,166 @@
+.. _multi_tenant_tutorial:
+.. highlight:: sql
+
+Multi-tenant data model
+########################
+
+In this tutorial, we will use a sample ad analytics dataset to demonstrate how you can       
+use Citus to power your multi-tenant application.                             
+
+.. note::
+                                                                                             
+    This tutorial assumes that you already have Citus installed and running. If you don't have Citus running,
+    you can provision a cluster using `Citus Cloud <https://console.citusdata.com>`_ or setup Citus locally
+    using :ref:`single_machine_docker`.
+
+
+Data model and Schema
+---------------------
+
+We will demo building the database for an ad-analytics app which companies can use to view, change,
+analyze and manage their ads and campaigns (see an `example app <http://citus-example-ad-analytics.herokuapp.com/>`_).
+To represent this data, we will use three Postgres tables:- :code:`companies`, :code:`campaigns` and :code:`ads`.                                                                
+
+Such an application and data model have good characteristics of a typical multi-tenant system. Data from different tenants is stored in a central database, and each tenant has an isolated view of their own data.
+                                                                                             
+To get started, you can first connect to the Citus co-ordinator using psql.
+
+**If you are running Citus Cloud**, you can connect by specifying the connection string (URL in the formation details):
+    
+::
+    
+    psql connection-string
+
+**If you are running on Docker**, you can connect by running psql with the docker exec command:
+
+::
+    
+    docker exec -it citus_master psql -U postgres
+
+Then, you can create the tables by using standard PostgreSQL :code:`CREATE TABLE` commands.
+
+::
```

Comment:
Small note - the code blocks render better in sphinx if you label them with a language. In this case you could do

```rst
.. code-block:: sql
```

rather than the bare double colon. Syntax highlighting is supported for [these languages](http://pygments.org/languages/).